### PR TITLE
Update ballerina 'build', 'clean' and 'search' command cli help

### PR DESCRIPTION
**Pull request by SupunS**
## Purpose
$subject.

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
-           Skips test compilation and execution.
+           Skip test compilation and execution.
 
        --skip-lock
            Skip using the lock file to resolve dependencies.
 
+       --experimental
+           Enable experimental language features.
+
+       --config
```
Comment (rdhananjaya) [inline cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help]: There seems to be a new config format https://github.com/ballerina-platform/ballerina-lang/pull/18708/files#diff-fd96dc1ef9c62010f9406cc6373ea3f4R50

Comment (SupunS) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help]: This is a CLI option expects by the build command.
Looks like the old style config passing has not being removed form the build command.
Shall we fix it in a separate PR?

Comment (rdhananjaya) [reply cli/ballerina-tool/src/main/resources/cli-help/ballerina-build.help]: +1

**Comment by sameerajayasoma** [comment]
Merged to the fix-help-text branch

**Pull request closed without merging**